### PR TITLE
Enhance log messages

### PR DESCRIPTION
**Pull request by gcollazo**
* Default message stays the same
* Groups logs using `console.groupCollapsed`
* Adds request to logged data
* Adds "raw" request to logged data

Demo: https://www.youtube.com/watch?v=qV_HlRnvKZk

### CODE REVIEW

**Comment by jherdman** [comment]
This is awesome.

File: addon/server.js [hunk 1]
```diff
@@ -38,17 +38,30 @@ function createPretender(server) {
 
     this.handledRequest = function(verb, path, request) {
       if (server.shouldLog()) {
-        console.log(`Mirage: [${request.status}] ${verb.toUpperCase()} ${request.url}`);
-        let { responseText } = request;
-        let loggedResponse;
+        console.groupCollapsed(
```
Comment (jherdman) [inline addon/server.js]: Can you confirm that this works on mobile Safari and mobile Chrome? Compatibility [isn't clear](https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed), and given that one may need to run tests against these targets it'd be nice to not have surprise fails here.

**Comment by gcollazo** [comment]
Thanks for the feedback @jherdman. Did some testing and it works on all the tested devices. Do you think we need to test on more devices?

**ios7**
ios7

**ios11**
ios11

**android4.4**
android4 4

**android7**
android7

**Comment by samselikoff** [comment]
Very cool @gcollazo – thanks!!

**Pull request merged**